quickstarts/rhdh-installation-via-operator.yaml

```diff
@@ -139,26 +142,164 @@ spec:
       description: |-
         To create a Red Hat Developer Hub instance, you should create a `Backstage` custom resource.
 
-        1. If needed, switch or create a new project from the **Project dropdown**.
-        2. Click on the [perspective switcher]{{highlight qs-perspective-switcher}} at the top of the navigation and select **Administrator**.
-        3. In the Administrator perspective, go to the **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation.
+        1. Click on the [perspective switcher]{{highlight qs-perspective-switcher}} at the top of the navigation and select **Administrator**.
+        2. In the Administrator perspective, go to the **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation.
+        3. If needed, switch or create a new project from the **Project dropdown**.
         4. Select the "Red Hat Developer Hub" operator and navigate to the **Backstage** tab.
```

Comment:
I think the tab is "Red Hat Developer Hub":

```suggestion
        4. Select the "Red Hat Developer Hub" operator and navigate to the **Red Hat Developer Hub** tab.
```

![image](https://github.com/openshift/console-operator/assets/593208/f4818429-2ac7-451e-9c04-fa2d394a2e95)

Reply:
Oh thanks for that hint. I expected that the Tab shows the resource name. So that the Tab, the title below and the create button on the right side (Create Backstage) shows the same "title".